Makefile

```diff
@@ -170,6 +175,10 @@ golangci-lint:
 .PHONY: test
 test: manifests generate gowork fmt vet envtest ginkgo ginkgo-run ## Run ginkgo tests with dependencies.
 
+.PHONY: test-ci
+test-ci: GINKGO_ARGS = $(GINKGO_CI_ARGS)
```

Comment:
do we need a new target for this? can't we just set different  GINKGO_ARGS in the CI job?

Reply:
That was my original suggestion. But it was pointed out that we would have to submit PR to release repo every time we would want to change anything about the way ginkgo runs, say number of repetitions.

Reply:
ah ok, so maybe we should get some review of those who requested it?

Reply:
They are on the list.